tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "./node_modules/@dojo/scripts/tsconfig/commonjs.json",
 	"compilerOptions": {
+		"skipLibCheck": true,
```

Comment:
Needed as currently using intern 4.3, this is fixed in 4.8 which needs to be updated in dojo/scripts